tests/integrationv2/utils.py

```diff
@@ -76,18 +76,6 @@ def invalid_test_parameters(*args, **kwargs):
         if not provider_.supports_protocol(protocol):
             return True
 
-    if provider is not None and other_provider is not None:
```

Comment:
If these exceptions are no longer needed, can this PR close some of these issues: https://github.com/aws/s2n-tls/issues/3246

Reply:
It can! I've added a "resolves" to this PR for both the fips-related ones.